openlibrary/macros/StarRatingsStats.html

```diff
@@ -4,9 +4,9 @@
 $ stats_by_bookshelf = work and work.get_num_users_by_bookshelf() or {}
 $ ratings_average = rating_stats.get('avg_rating', None)
 $ ratings_count = rating_stats.get('num_ratings', None)
-$ want_to_read_count = "{:,}".format(stats_by_bookshelf.get('want-to-read', None))
-$ currently_reading_count = "{:,}".format(stats_by_bookshelf.get('currently-reading', None))
-$ already_read_count = "{:,}".format(stats_by_bookshelf.get('already-read', None))
+$ want_to_read_count = stats_by_bookshelf.get('want-to-read') and "{:,}".format(stats_by_bookshelf['want-to-read'])
```

Comment:
We couldn't just default to empty string in the `get` because this would result in `ValueError: Cannot specify ',' with 's'.`